covid-vaccine-faq-bot/README.md

```diff
@@ -19,7 +19,7 @@ agent that interacts with the user. You will need to setup a Google Cloud projec
 
 ### 2. Create a Dialogflow Chatbot Agent
 
-The conversational chatbot is powered by  an integration between Twilio and Google Dialogflow, a natural language understanding platform used to design and integrate a conversational user interfaces. You may import the pre-built Dialogflow chatbot agents included in this repository, or configure your own chatbot agent.
+The conversational chatbot is powered by  an integration between Twilio and Google Dialogflow, a natural language understanding platform used to design and integrate a conversational user interfaces.You may import the pre-built Dialogflow chatbot agents included in this repository, or configure your own chatbot agent.
```

Comment:
I think you removed the space from the wrong place. I meant between the "by" and "an" but realized that doesnt make a diff in the render. Taking a space away here, does make a difference so I think you should put it back.

Reply:
Fixed.